02_bubble/madhavansu/main_test.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote(`[-2092 -10 -5 -2 -2 0 2 3 9 19 29][-2092 -10 -5 -2 -2 0 2 3 9 19 29]`)
+	actual := strconv.Quote(buf.String())
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+func TestBubbleAndInsertion(t *testing.T) {
+
```

Comment:
Remove blank lines at beginning/end of block.

Reply:
Removed.